airbyte-commons/src/test/java/io/airbyte/commons/json/JsonsTest.java

```diff
@@ -24,27 +24,34 @@
 
 package io.airbyte.commons.json;
 
+import static org.junit.jupiter.api.Assertions.assertArrayEquals;
```

Comment:
intellij did this for me. i'm keeping it though since it's more in line with how our other tests are written anyway.